src/Avalonia.FreeDesktop/DBusSystemDialog.cs

```diff
@@ -0,0 +1,80 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Avalonia.Controls;
+using Avalonia.Controls.Platform;
+using Tmds.DBus;
+
+namespace Avalonia.FreeDesktop
+{
+    public class DBusSystemDialog : ISystemDialogImpl
```

Comment:
It should be used here optionally (depending on X11PlatformOptions property) - https://github.com/AvaloniaUI/Avalonia/blob/82c7d7d64f91a485fb05ab0396d53caf9556f72a/src/Avalonia.X11/X11Platform.cs#L83

Reply:
But as you mentioned, it's better to just replace old one with this as for me.
Let's wait for @jmacato and @kekekeks opinion

Reply:
We need some kinda of a gracefull fallback in case DBus or xdg-desktop-portal are not available. Might as well fall back to the managed dialog if GTK is not available as well.